hammer/par/openroad/__init__.py

```diff
@@ -1043,14 +1394,43 @@ def setup_klayout_techfile(self) -> bool:
                 self.logger.info("Modifying Klayout Techfile: {} -> {}".format
                     (source_path, dest_path))
                 for line in sf:
+                    if "<lef-files>merged.lef</lef-files>" in line:
+                        continue
                     if '</lefdef>' in line:
                         df.write(insert_lines)
                     df.write(line)
-        return True
+        return str(klayout_techfile_path)
+
+
+    def remove_pcells_from_netlist(self):
```

Comment:
Wow, this is impressive but incredibly hacky. OpenROAD can't exclude instances of some cells in netlist writing? Can the physical only cells stuff remove this sky130 hardcoding?

Reply:
Just removed this, it was fixed in a new version of openroad but I had used this in the meantime to check LVS.